### PR TITLE
Data sta rmap cortex tiesmeyer, folder creation changed

### PR DESCRIPTION
**Pull request by sebastiantiesmeyer**

### CODE REVIEW

File: data/STARmap-2018-mouse-cortex/prepare_data.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            df_coordinates_tsv.columns=[['x','y']]
+            df_coordinates_tsv.index = adata.obs.index
+            print(f"Coordinates: {df_coordinates_tsv.head()}")
+            df_coordinates_tsv.to_csv(sample_output_folder/'coordinates.tsv',sep="\t",index_label="")
+        
+            # Matrix
+            scipy.io.mmwrite(f"{sample_output_folder}/counts.mtx",adata.X)
+        
+            # Anndata
+            adata.write_h5ad(f"{sample_output_folder}/anndata.h5ad")
```
Comment (niklasmueboe) [inline data/STARmap-2018-mouse-cortex/prepare_data.py:88]: Do we want to keep the h5ad file?

File: data/STARmap-2018-mouse-cortex/prepare_data.py
Comment (niklasmueboe) [inline data/STARmap-2018-mouse-cortex/prepare_data.py:1]: I think we mostly used the convention to have the same name for the file and folder i.e. STAR,ap-2018-mouse-cortex

File: data/STARmap-2018-mouse-cortex/prepare_data.py (hunk 1 above)
Comment (sebastiantiesmeyer) [reply data/STARmap-2018-mouse-cortex/prepare_data.py:88]: You decide. Probably cleaner to remove it.

Comment (niklasmueboe) [reply data/STARmap-2018-mouse-cortex/prepare_data.py:88]: Then let's not generate unnecessary data for now

File: data/STARmap-2018-mouse-cortex/prepare_data.py
Comment (sebastiantiesmeyer) [reply data/STARmap-2018-mouse-cortex/prepare_data.py:1]: you mean prepare_data.py? So rename 
prepare_data.py -> STAR,ap-2018-mouse-cortex.py
?

Comment (niklasmueboe) [reply data/STARmap-2018-mouse-cortex/prepare_data.py:1]: yes

File: data/STARmap-2018-mouse-cortex/prepare_data.py (hunk 1 above)
Comment (sebastiantiesmeyer) [reply data/STARmap-2018-mouse-cortex/prepare_data.py]: fixed with new push

File: data/STARmap-2018-mouse-cortex/prepare_data.py
Comment (sebastiantiesmeyer) [reply data/STARmap-2018-mouse-cortex/prepare_data.py:1]: fixed with new push

**Review by niklasmueboe — APPROVED** [review]

**Pull request merged**